tests/AutoReview/ProjectCodeTest.php

```diff
@@ -331,6 +331,10 @@ function ($item) {
 
     public function provideDataProviderMethodNameCases()
     {
+        if (extension_loaded('xdebug')) {
+            $this->markTestSkipped('Data provider too slow when Xdebug is loaded.');
```

Comment:
can we detect if we're on a CI here and than throw an exception? accidentally skipping the tests on CI is unwanted I think

Reply:
I think none of our CI builds use Xdebug so is this really needed?

Reply:
[Travis CI](https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables), [AppVeyor](https://www.appveyor.com/docs/environment-variables/) and [CircleCI](https://circleci.com/docs/1.0/environment-variables/) all define a `CI` environment variable so I updated my PR to not skip the test when this variable exists.

Reply:
thanks :)  good to ship!